internal/state/change_processor.go

```diff
@@ -104,7 +104,10 @@ func (c *ChangeProcessorImpl) CaptureUpsertChange(obj client.Object) {
 	c.changed = c.changed || c.store.changed || c.cfg.RelationshipCapturer.Exists(obj, client.ObjectKeyFromObject(obj))
 }
 
-func (c *ChangeProcessorImpl) CaptureDeleteChange(resourceType client.Object, nsname types.NamespacedName) {
+func (c *ChangeProcessorImpl) CaptureDeleteChange(
```

Comment:
What is the rationale for splitting the arguments across multiple lines here? (there are a few more similar changes). It didn't run over 120 characters. Is it a proposal for a new style - any function with more than one arguments must have them defined across multiple lines?

Reply:
Shoot. No, I had an issue where my IDE line linter was set to 100 characters instead of 120, and any file that I touched manually was linted. I thought I caught all of them. Are these changes limited to this file?

Reply:
yep, that file